packages/mobile/src/services/offline-downloader/offline-download-queue.ts

```diff
@@ -197,29 +197,28 @@ export const cancelQueuedDownloads = async (
   )
   queue.stop()
   const jobs = await queue.getJobs()
-  const jobsToCancel = jobs.filter(({ workerName, payload }) => {
+  jobs.forEach(async (rawJob) => {
```

Comment:
here too just in case you we dont need

Reply:
great eye! not necessary. i copied from code before too, so win-win! thx for thorough rev